src/sidebar/components/SelectionTabs.js

```diff
@@ -136,7 +138,16 @@ function SelectionTabs({ isLoading, settings }) {
         )}
       </div>
       {selectedTab === 'note' && settings.enableExperimentalNewNoteButton && (
-        <NewNoteBtn />
+        <div className="u-layout-row--justify-right">
+          <LabeledButton
+            icon="add"
+            onClick={() => annotationsService.createPageNote()}
+            variant="primary"
+            style={applyTheme(['ctaBackgroundColor'], settings)}
+          >
+            New note
```

Comment:
Have you considered in title casing?

```suggestion
            New Note
```

Surrounding elements have title casing:
![image](https://user-images.githubusercontent.com/8555781/113295058-fdf1ed00-92f7-11eb-84d6-e6e7ea62436b.png)

Reply:
Our casing conventions for buttons is usually sentence casing. _Public_ is capitalized in _Post to Public_ because it's the name of a group. The "Page Notes" tab actually is the outlier here. e.g. other buttons use sentence casing:

![image](https://user-images.githubusercontent.com/439947/113306342-fcadc980-92d1-11eb-83f5-7faaa1482409.png)

I left casing as-is on this button. But your point is interesting: Is _Page Notes_ effectively a "proper noun" in this usage, meaning that _Notes_ should be capitalized?